### DISCUSSION TITLE
Change name and image

### DISCUSSION THREAD

**Discussion by scorine99** [OrcaSlicer/OrcaSlicer#5325] [Q&A]
How do I change these on my OrcaSlicer 2.0
![Screenshot 2024-05-11 004247](https://github.com/SoftFever/OrcaSlicer/assets/168488744/a6643d7c-77ed-4799-aa3f-98c0fa6f9bd0)

**Comment by sworcester001** [comment]
That looks like your Bambu cloud login name
